<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1111</width>
      <height>1200</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QWidget" name="media_widget" native="true">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <widget class="QGroupBox" name="local_box">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>300</width>
        <height>600</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <widget class="LocalVideoWidget" name="local_video" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>280</width>
         <height>250</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>280</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
      <widget class="LocalAudioWidget" name="local_audio" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>290</y>
         <width>280</width>
         <height>50</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="camera_box">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>350</y>
         <width>140</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>350</y>
         <width>140</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>摄像头设备:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="audio_box">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>390</y>
         <width>140</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>390</y>
         <width>140</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>音频设备:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="audio_button">
       <property name="geometry">
        <rect>
         <x>165</x>
         <y>430</y>
         <width>120</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>启动（麦克风）</string>
       </property>
      </widget>
      <widget class="QPushButton" name="video_button">
       <property name="geometry">
        <rect>
         <x>25</x>
         <y>430</y>
         <width>120</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>启动（摄像头）</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>480</y>
         <width>30</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icons.qrc">:/image/icons/next1.bmp</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>520</y>
         <width>30</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icons.qrc">:/image/icons/pre1.bmp</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QSlider" name="audioinput_slider">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>480</y>
         <width>220</width>
         <height>30</height>
        </rect>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="audiooutput_slider">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>520</y>
         <width>220</width>
         <height>30</height>
        </rect>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="vs_box">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>600</y>
        <width>300</width>
        <height>600</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_runtime">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>440</y>
         <width>140</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-left: 1px solid red;

border-right: 1px solid red;

border-top: 1px solid red;

border-bottom: 1px solid red;
</string>
       </property>
       <property name="text">
        <string>运行时间</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_status">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>440</y>
         <width>140</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
border-left: 1px solid red;

border-right: 1px solid red;

border-top: 1px solid red;

border-bottom: 1px solid red;
</string>
       </property>
       <property name="text">
        <string>状态</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QWidget" name="vs_video" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>280</width>
         <height>280</height>
        </rect>
       </property>
      </widget>
      <widget class="LocalAudioWidget" name="vs_audio" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>320</y>
         <width>280</width>
         <height>50</height>
        </rect>
       </property>
      </widget>
      <widget class="QSlider" name="audiooutput_vs">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>380</y>
         <width>200</width>
         <height>30</height>
        </rect>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>380</y>
         <width>30</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icons.qrc">:/image/icons/pre1.bmp</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>380</y>
         <width>30</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/image/icons/check2.ico</normaloff>:/image/icons/check2.ico</iconset>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_group">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="mwidget" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <layout class="QHBoxLayout" name="status_layout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="label_global_runtime">
         <property name="text">
          <string>运行时间</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_global_status">
         <property name="text">
          <string>状态</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_global_number">
         <property name="text">
          <string>链接数量</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_global_delay">
         <property name="text">
          <string>延迟</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1105</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumode">
    <property name="title">
     <string>模式</string>
    </property>
    <widget class="QMenu" name="menuMode">
     <property name="title">
      <string>模式</string>
     </property>
     <addaction name="actionserver_mode"/>
     <addaction name="actionclient_mode"/>
    </widget>
    <addaction name="action_launch"/>
    <addaction name="separator"/>
    <addaction name="menuMode"/>
    <addaction name="separator"/>
    <addaction name="action_setting"/>
    <addaction name="separator"/>
    <addaction name="action_close"/>
   </widget>
   <widget class="QMenu" name="menucontrol">
    <property name="title">
     <string>控制</string>
    </property>
    <addaction name="action_linkInfo"/>
    <addaction name="separator"/>
    <addaction name="action_blackList"/>
    <addaction name="separator"/>
    <addaction name="action_whiteList"/>
   </widget>
   <widget class="QMenu" name="menuinformation">
    <property name="title">
     <string>信息</string>
    </property>
    <addaction name="action_time"/>
    <addaction name="separator"/>
    <addaction name="action_linke"/>
    <addaction name="separator"/>
    <addaction name="action_status"/>
   </widget>
   <widget class="QMenu" name="menuinfo">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menumode"/>
   <addaction name="menucontrol"/>
   <addaction name="menuinformation"/>
   <addaction name="menuinfo"/>
  </widget>
  <action name="action_launch">
   <property name="text">
    <string>启动</string>
   </property>
  </action>
  <action name="action_close">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
   <property name="toolTip">
    <string>关于???</string>
   </property>
  </action>
  <action name="action_setting">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="actionserver_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>服务器模式</string>
   </property>
  </action>
  <action name="actionclient_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>客户端模式</string>
   </property>
  </action>
  <action name="action_blackList">
   <property name="text">
    <string>黑名单</string>
   </property>
  </action>
  <action name="action_whiteList">
   <property name="text">
    <string>白名单</string>
   </property>
  </action>
  <action name="action_linkInfo">
   <property name="text">
    <string>链接信息</string>
   </property>
  </action>
  <action name="action_time">
   <property name="text">
    <string>时间</string>
   </property>
  </action>
  <action name="action_linke">
   <property name="text">
    <string>链接</string>
   </property>
  </action>
  <action name="action_status">
   <property name="text">
    <string>状态</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LocalVideoWidget</class>
   <extends>QWidget</extends>
   <header>Widget/localvideowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LocalAudioWidget</class>
   <extends>QWidget</extends>
   <header>Widget/localaudiowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_close</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
